public/js/updateInfo.js

```diff
@@ -47,9 +48,8 @@ const updateNameHandler = async (event) => {
     // Would be better to eventually use something other than an alert
     if (response.ok) {
         alert('Name updated successfully');
-        document.location.replace('/profile');
```

Comment:
To prevent an updating-user-info bug, the page needs to be re-rendered _only if the user is updating their email._